shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -107,7 +125,6 @@ @implementation FlutterViewController {
   // https://github.com/flutter/flutter/issues/35050
   fml::scoped_nsobject<UIScrollView> _scrollView;
   fml::scoped_nsobject<UIHoverGestureRecognizer> _hoverGestureRecognizer API_AVAILABLE(ios(13.4));
```

Comment:
It's a bit stranging seeing only one gesture recognizer using `scoped_nsobject`. Can you convert this one to a property too?